lib/engine/game/g_1862_solo/game.rb

```diff
@@ -0,0 +1,215 @@
+# frozen_string_literal: true
+
+require_relative '../g_1862/game'
+require_relative 'corporation'
+require_relative 'meta'
+
+module Engine
+  module Game
+    module G1862Solo
+      class Game < G1862::Game
+        include_meta(G1862Solo::Meta)
+
+        attr_reader :ipo_rows
+
+        # No cert limit
+        CERT_LIMIT = {
+          1 => 999,
+        }.freeze
+
+        STARTING_CASH = {
+          1 => 600,
+        }.freeze
+
+        # No cert limit
+        def show_game_cert_limit?
+          false
+        end
+
+        def setup_preround
+          @original_corporations = @corporations.dup
+
+          super
+
+          @original_corporations.reject { |c| @corporations.include?(c) }.each do |c|
+            hex = @hexes.find { |h| h.id == c.coordinates } # hex_by_id doesn't work here
```

Comment:
Out of curiosity, why doesn't this work here?

Reply:
@roseundy No idea - this code was copied from g_1862/game.rb and the comment is there in the code.

Reply:
That's funny. I must have written it :-)